README.md

```diff
@@ -1,2 +1,4 @@
 # sfosc-roles-experimentation-ground
 Playground to find a nice setup to match SFOSC governance and ideas.
```

Comment:
Use the word "Training Area". It sounds more serious.

Reply:
I was thinking a badge for that extra serious legitimacy.